File: inflate/tests/langs/zig/box.zig
const Box = struct {
    size: i32 = 1,

    fn helper(self: Box, value: i32) i32 {
        return value + self.size;
    }

    fn greet(self: Box, value: i32) i32 {
        return self.helper(value) * 2;
    }

    fn runner(self: Box) i32 {
        return self.greet(3);
    }
};

File: inflate/tests/langs/rust/box.rs
pub struct Box {
    size: i32,
}

impl Box {
    fn helper(&self, value: i32) -> i32 {
        value + self.size
    }

    fn greet(&self, value: i32) -> i32 {
        self.helper(value) * 2
    }

    fn runner(&self) -> i32 {
        self.greet(3)
    }
}
